ci/qa/phpstan-baseline.php

```diff
@@ -1016,11 +1016,6 @@
 	'count' => 1,
 	'path' => __DIR__ . '/../../src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php',
 ];
-$ignoreErrors[] = [
-	'message' => '#^Right side of && is always true\\.$#',
-	'count' => 1,
-	'path' => __DIR__ . '/../../src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator.php',
-];
```

Comment:
Did PHPStan drop this sniff? I see no change to the `JsonGenerator` in this changeset to warrant the removal of these lines. :man_shrugging:

Reply:
maybe phpstan got smarter, or an upstream library changed.

The other change, `if (!isset($organizationCodeMatches[1])) {` suddenly was an issue that phpstan wanted fixed.